### PR TITLE
Fix URL typos in Cargo.toml

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name = "wayland-kbd"
 version = "0.2.0"
 documentation = "http://vberger.github.io/wayland-kbd/wayland_kdb/"
```
Comment (mbrubeck) [inline Cargo.toml]: The documentation link has a typo too.

Comment (frewsxcv) [inline Cargo.toml]: https://github.com/vberger/wayland-kbd/pull/2

Comment (frewsxcv) [inline Cargo.toml]: I suppose I could just modify this `patch-1` branch, but that sounds like a lot of work :)

Comment (frewsxcv) [inline Cargo.toml]: Fixed in latest force push

**Comment by elinorbgr** [comment]
Didn't notice it, thanks! :smile:

**Pull request merged**